liteflow-labs/starter-kit
# Use proper img component

### Project organization

- Dependant on https://github.com/liteflow-labs/starter-kit/pull/160

### Description

Implement image size everywhere.
- Used the `layout="fixed"` with `width` and `height` whenever the image is display as a square with always the same size
- Implement media query in `sizes` prop otherwise
- Use the custom Image component everywhere

### How to test

- Navigate on the different page
- Check the image loaded are not full width, hopefully very close to the rendered size (most of the time it's close to 2 size of the rendered size because of pixel density)

### Checklist
